test/e2e/bundesident.spec.ts

```diff
@@ -22,7 +22,10 @@ describe("Identifikation option", () => {
         "user-agent": mobileUserAgent,
       },
     });
-    cy.contains("dt", "Identifikation mit Ihrem Ausweis");
+    cy.contains(
+      "h1",
+      "Möchten Sie sich in wenigen Minuten mit Ihrem Ausweis identifizieren?"
+    );
```

Comment:
I think this is no longer testing what it was before because we now redirect. Imo we should click on the _Weiter_ button and then test on the actual /identifikations page